R/maraca.R

```diff
@@ -347,124 +430,109 @@ plot_maraca <- function(
     ) +
     ggplot2::guides(fill = "none")
 
+  # Add label to plot - maracaPlot
+  attr(plot, "class") <- c(attr(plot, "class"), "maracaPlot")
```

Comment:
It's the other way around. Order of resolution is first to last, not last to first.

Reply:
also use class(x) <- c("maracaPlot, class(x))